src/utils/constants.ts

```diff
@@ -151,7 +151,7 @@ export const DEFAULT_CDD_ID = '0x00000000000000000000000000000000000000000000000
 /**
  * Minimum version of Middleware V2 GraphQL Service (SubQuery) that is compatible with this version of the SDK
  */
-export const MINIMUM_SQ_VERSION = '18.0.2';
+export const MINIMUM_SQ_VERSION = '19.3.0';
```

Comment:
What would fail if SQ is < 19.3.0?

Reply:
Few enums and a warning will be thrown out